guides/common/modules/proc_adding-a-bmc-interface.adoc

```diff
@@ -30,15 +29,30 @@ ifndef::satellite[]
 . On the *Proxies* tab, select your {SmartProxy} as *BMC Proxy*.
 endif::[]
 . Click *Submit*.
-. Navigate to *Hosts* > *All Hosts*.
-. Click *Edit* next to the host you want to edit.
-. On the *Interfaces* tab, click *Add Interface*.
+
+. Navigate to the *Add Interface* form:
++
+--
+* Navigate to *Hosts* > *All Hosts*.
```

Comment:
AFAIK, we use numbering, not bullets, for substeps.

Reply:
I'll default to complying with the rest of the docs for consistency, but I'll present my logic just in case:

* The usual option results into the `a. b. c.` numbering, which is counterintuitive (looks like answer options in a test)
* It's possible to do the better `i. ii. iii.` numbering, but the markup is illogical (`.` for level one items, `...` for level two items)
* I chose bullet points, because the markup is clean enough, and the result is clear. Plus, most of the grouped steps can be done in any order, it's a form in the UI.

I'll wait for a :+1: from another reviewer to change these to `a. b. c.`.

Reply:
My point of view is based on "separating content from presentation": you want to add substeps, so let's use the markup for substeps. The way that markup is rendered is a separate issue.